src/decode.h

```diff
@@ -240,6 +240,7 @@ typedef struct {
 
     /* instruction length */
     uint8_t insn_len;
+    bool is_tail;
```

Comment:
`is_tail` is a bit confusing. Someone might think it as "tail" in queue data structure. How about `bool tailcall`?

Reply:
Add informative comment for newly-added member.